### ISSUE TITLE
Prince of Persia 2 causes crash

### ISSUE THREAD

**Issue by jschwartzenberg** [stsp/fdpp#110]
**Describe the bug**
There is a crash, dosemu is terminated, `Terminated with signal 4`.

**To Reproduce**
Get in-game and play a bit.

**Attach the binaries or provide an URL**
http://www.abandonia.com/en/games/51/Prince+of+Persia+2+-+The+Shadow+%2526+The+Flame.html

**Attach the log**
[boot.log](https://github.com/stsp/dosemu2/files/3775198/boot.log)

Same game as stsp/dosemu2#290. This issue is new, needs a bisect.

**Comment by stsp** [comment]
Looks fdpp-specific.

**Comment by stsp** [comment]
And again, I can reproduce this not on all
machines... But maybe depends on prince
version or its config too.

**Comment by jschwartzenberg** [comment]
I did not have my crash with fdpp. I think that's your (KVM) crash. This issue should be a dosemu2 issue and you should create a new one for your (KVM) crash ;)

At the same time, I think we should come up with a way to quickly compare a given dosemu2 configuration to another based on relevant basic info. Info such as dos version, kvm/cpuemu, linux version, maybe distro?, 32-bit/64-bit, cpu etc. This can probably all be deduced from `boot.log` but it is not so easy to use for comparison.

**Comment by stsp** [comment]
Mine is not even related to kvm.
I have prince2+fdpp crashing on
one machine with any config.
Something very wrong is happening
if we started to get so different results.

Log is insufficient. What we need is
a way to quickly share dosemu binaries+
configs+setup+game. Currently this
is too scattered to share quickly.
So far I dont remember many examples
of a broken build (except when ld
started to corrupt binaries), so perhaps
the binaries can be excluded. But
we need a way to bundle
drives+config+game. One way of
doing that, is to stop installing
dosemu.conf globally and insist
on local configs. Then ~/.dosemu
will contain everything we need,
including log, and can be easily
tarred up.
I really don't see the need for the
global, root-only-writable config file
these days.

**Comment by jschwartzenberg** [comment]
Yes, but yours is different from mine in that it happens with and without KVM. Mine only happens with CPUEMU. Sharing a structure seems like a good idea. I like the idea of getting rid of `/etc/dosemu/dosemu.conf` and generating `~/.dosemurc` when it doesn't exist on start-up :)

**Comment by stsp** [comment]
You can of course open another
ticket for prince2, but reporting
something I can't reproduce, has
a little value unless you can provide
more info. Another possibility is to
wait until this one is solved, re-check
the problem and open if its not fixed.

**Comment by jschwartzenberg** [comment]
I would expect you could reproduce this my problem with FreeDOS & CPUEMU.
